tests/loop.cpp

```diff
@@ -11,13 +11,13 @@ TEST_BOTH(01_record_loop) {
         for (uint32_t j = 0; j < 2; ++j) {
             UInt32 x = arange<UInt32>(10);
             Float y = zero<Float>(1);
-            Float z = 1;
+            Float z = Float(1);
```

Comment:
I am curious: why does this not work? Having an implicit conversion from an integer to a float is probably a good idea.

Reply:
This has been fixed